aliases/general.aliases.sh

```diff
@@ -28,3 +28,10 @@ alias show_options='shopt'                  # Show_options: display bash options
 alias fix_stty='stty sane'                  # fix_stty:     Restore terminal settings when screwed up
 alias cic='set completion-ignore-case On'   # cic:          Make tab-completion case-insensitive
 alias src='source ~/.bashrc'                # src:          Reload .bashrc file
+
+# Outputs List of Loadable Modules (llm) for current kernel
+alias llm="find /lib/modules/$(uname -r) -type f -name '*.ko*'"
```

Comment:
@Kreyren I think these aliases probably go to misc.aliases.sh which make more sense than put it under general? What do you think?

Reply:
I don't remember :thinking: Use your best judgement?

Reply:
@Kreyren Probably go to misc 😄  General should be kept as minimal as possible or something can affect daily tasks and not very specified to any OSes